proposals/interface-types/working-notes/choices/webidl.md

```diff
@@ -0,0 +1,270 @@
+# Interface Types and WebIDL
+
+## Preamble
+This note is part of a series that explain some of the major design choices in
+the Interface Types specification.
+
+## Introduction
+
+A key motivation for Interface Types is the semantic gap between the
+expressivitiy of core WebAssembly and the needs of API designers and application
+authors. Part of this gap is represented by the difference between the type
+system of core WebAssembly and the needs of a typical programmer.
+
+As an example of this, core WebAssembly has no intrinsic type corresponding to
+`string`; even though this is arguably the most important type in many if not
+most applications.
+
+Note that this is not an oversight. Neither the MVP of WebAssembly nor the
+vision implied by the [GC proposal][GC] have `string` as an intrinsic type.
+
+A core part of this proposal is the design of an IDL (Interface Definition
+Language); one goal of which is to address this semantic gap. However, in
```

Comment:
I would feel more comfortable saying "A core part of this proposal is to define a new set of types which address this semantic gap, allowing WebAssembly function signatures to use high-level types like `string` directly, without fixing a core WebAssembly representation.  This goal overlaps with the goals of traditional IDLs (Interface Definition Languages), leading to the following questions."  In particular, I do not think we should say we're designing an IDL.

Reply:
We ARE designing an IDL though.